spec/system/scroll_manager_service_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+describe "Ember route-scroll-manager service", type: :system do
+  before do
+    Fabricate(:admin)
+    50.times { Fabricate(:post) }
+  end
+
+  let(:discovery) { PageObjects::Pages::Discovery.new }
+  let(:topic) { PageObjects::Pages::Topic.new }
+
+  def current_scroll_y
+    page.execute_script <<~JS
+      return window.scrollY
+    JS
```

Comment:
🏌️, I think

```suggestion
    page.evaluate_script("window.scrollY")
```

Reply:
The `return` is important here. Plus, I kinda like the heredoc because it means we get JS syntax highlighting

Reply:
`return` isn't needed when using `evaluate_script` (which is for cases where you need to return a value)

Reply:
Oooh I didn't spot the `execute`/`evaluate` change 👌